src/assets/sass/main/component/styles.scss

```diff
@@ -2,6 +2,10 @@
 
 $site-wrapper-padding-top: 112px;
 
+html, body {
```

Comment:
This solution is not working. The langiage-switch is still overflowing. Try this: 

```css
html, body {
  overflow-x: hidden;
  max-width: 100%;
  position: relative;
}
```